tests/test_warc_to_zim.py

```diff
@@ -168,6 +168,9 @@ def verify_warc_and_zim(self, warcfile, zimfile, verify_scraper_suffix):
                 # have exact match
                 if payload.mimetype.startswith("text/html"):
                     assert head_insert in payload_content
+            elif record.rec_type == "resource":
+                # we do not want to embed resources "as-is"
+                assert not payload
```

Comment:
Should we really fail if a resource has a payload ?
It open a way to have a change in a part we don't use breaking our code.

Reply:
This is a test 👀

Reply:
And the test says: "we do not produce a ZIM item for WARC record of type `resource`", which must be true for now ; might change in the future if we decide to use `resource` records, but then we will fix the test intentionally because we know which kind of `resource` must produce which kind of `payload` ; and I even doubt we will embed them at the same path than the one used in WARC

Reply:
> This is a test 👀

🤦🏽 My bad. We are good then.